index.js

```diff
@@ -0,0 +1,18 @@
+document.querySelector('button').addEventListener('click', async () => {
+  const button = document.getElementById('boom');
+  button.style.height = '150px';
+  button.style.width = '200px';
+
+	await Tone.start()
+  //create a synth and connect it to the master output (your speakers)
+  var synth = new Tone.Synth().toMaster()
+
+  //play a middle 'C' for the duration of an 8th note
+  synth.triggerAttackRelease('C4', '8n')
+})
+
+document.querySelector('button').addEventListener('mouseout', async () => {
+  const button = document.getElementById('boom');
+  button.style.height = '100px';
+  button.style.width = '150px';
+});
```

Comment:
Like Monarch showed on Twicht, here you could create one general function and user different variables for both event listeners. It would make your code dryer.

```suggestion
const generalFunction = (el, evt, height, width) => {
document.querySelector(el).addEventListener(evt, async () => {
  const button = document.getElementById('boom');
  button.style.height = heigth;
  button.style.width = width;
});

generalFunction('button', 'click', '150px', '200px');
generalFunction('button', 'mouseout', '100px', '150px');
```